doc/cygnus-ngsi/flume_extensions_catalogue/ngsi_mysql_sink.md

```diff
@@ -210,6 +210,7 @@ If `attr_persistence=colum` then `NGSIMySQLSink` will persist the data within th
 | mysql\_port | no | 3306 ||
 | mysql\_username | no | root | `root` is the default username that is created automatically |
 | mysql\_password | no | N/A | Empty value as default (no password is created automatically) |
+| mysql\_maxPoolSize | no | 1 | Max number of connections per database pool (don't change this parameter) |
```

Comment:
Yo aquí elminaría la coletilla de "(don't change this parameter)". Ya hemos dicho que no pasa nada por tener un valor mayor. Y además me parece que ahora por defecto el valor es 2. Haria mencion además a las futuras capacidades de loadbalancing que se supone que van a hacer uso de esta funcionalidad